crates/cxx-qt-gen/src/parser/property.rs

```diff
@@ -162,25 +189,46 @@ impl ParsedQProperty {
                     ))
                 }
 
-                if let Some(read) = read {
+                let cxx_name_str = if let Some(name) = cxx_name {
+                    name.to_string()
+                } else if rust_name.is_none() {
+                    ident.to_string().to_case(Case::Camel)
+                } else {
+                    ident.to_string()
+                };
+                let mut name = Name::new(ident).with_cxx_name(cxx_name_str);
+
+                if let Some(ident) = rust_name {
+                    name = name.with_rust_name(ident);
+                }
```

Comment:
This feels like it belongs on `Name` itself, and should ideally share the logic with `Name::from_ident_and_attrs`.
Can we achieve that somehow?
That was kind of the point of having the `Name` type, to not have to reimplement this logic for eveything we parse.

Reply:
I think we did have a Name::property_name type method which did this but with the old auto camel case conversion, so I removed it. I can add a new method back which handles that